src/components/content/chart/utils.ts

```diff
@@ -21,22 +21,34 @@ function getEventsDict(events: any[], dict: any, timestampPropertyAccessor: (ele
   return getEventsMap(dict)
 }
 
+
+/**
+ *
+ * Aggregate the good and bad events datasets to minute-level counts for usage in a X,Y access of a Chart.js Bar.
+ * @export
+ * @param {any[]} goodEvents
+ * @param {any[]} badEvents
+ * @param {(element: any) => unknown} goodTimestampPropertyAccessor Accessor function for the good events timestamp we wish to use for ordering.
+ * @param {(element: any) => unknown} badTimestampPropertyAccessor Accessor function for the bad events timestamp we wish to use for ordering.
+ * @return {*, *}
+ */
 export function aggregateMicroDatasets(
   goodEvents: any[],
   badEvents: any[],
   goodTimestampPropertyAccessor: (element: any) => unknown,
   badTimestampPropertyAccessor: (element: any) => unknown
 ) {
 
-  var dict = Object()
+  var emptyChartDict = Object()
```

Comment:
You can also use `const` here together with the simple object declaration, no need for the constructor _(is not intuitive in JS)_

` const emptyChartDict = {};`;